internal/packager/registry.go

```diff
@@ -132,16 +135,16 @@ func Push(app *types.App, namespace, tag string) error {
 
 // Pull pulls an app from a registry
 func Pull(repotag string) error {
-	if err := pullImage(repotag); err != nil {
+	if err := pullImage(repotag, os.Stdout, os.Stderr); err != nil {
 		return err
 	}
-	return Load(repotag, ".")
+	return Load(repotag, ".", os.Stdout, os.Stderr)
 }
 
-func pullImage(repotag string) error {
+func pullImage(repotag string, stdout, stderr io.Writer) error {
```

Comment:
From a coding style point of view, I'll tend to put this kind of argument (stdOut/Err, context,...) as first arguments in the signature. @vdemeester ?

Reply:
Is there a style guide that spells that out that you could refer me to? Obviously Go has its own codes and I'm willing to become more familiar with them if only for harmony's sake, but that's a bit counterintuitive in my opinion.

The way I see it, this function has a main argument (`repotag`) which is directly being acted upon (the object). When you read the function's name, `pullImage`, the first question that comes to mind is "which image?", and that in my mind should be the first argument. `stdout` and `stderr` are very much auxiliary to the main purpose of the function. Refering to other languages that support optional parameters, this would be written as:
```c++
int pullImage(string repotag, ostream stdout = null, ostream stderr = null);
```
or
```python
def pull_image(repotag, stdout=None, stderr=None):
```

Reply:
With the move to `resto`, this has been removed anyway, but I'm still interested in doing some reading on the subject of Go coding style if you have something!

Reply:
There is some common go coding style, here is the part on the context: https://github.com/golang/go/wiki/CodeReviewComments#contexts